config.js

```diff
@@ -87,6 +87,20 @@ config.amqp = {
     options: {durable: true}
 };
 
+/**
+ * Configuration for the HTTP transport binding.
+ */
+config.http = {
+    /**
+     * South Port where the Ultralight transport binding for HTTP will be listening for device requests.
+     */
+    port: 7896,
```

Comment:
From where this default comes? Is the same that used by default if no configuration is provide (i.e. as it is currently at master branch)?

Reply:
I could not exactly understood what you wanted to ask ? Default in terms of what ?

Reply:
I'l try to make more explicit my question...

If IOTA-UL would run without specifying config.http.port (i.e. as config.js is now on master branch, where that element is not included), which port will be used?

Reply:
You mean to say if we remove http port from config.js then which port it will use by default. Is my understanding correct ?

Reply:
Yes, it is.

Bottom line is that it is unclear where the 7896 number comes from. Does it come from somewhere? Or did you pick it at random?

Reply:
The default HTTP port for IOTA is 7896. It is already mentioned in the docker-compose file of iotagent-json as an environment variable. You can verify it in the link below- https://github.com/telefonicaid/iotagent-json/blob/53c7d99cb680a123ec30152d0dae7674be724880/docker/README.md

Reply:
The current config.js file doesn't contain any configuration for http as it is in master which on starting the iotagent gives a configuration error as shown in the screenshot. 
![without_http](https://user-images.githubusercontent.com/42174460/51296918-332a0180-1a44-11e9-92db-5c72f1e7af8a.png) . But when we define http configuration as done in 
e2b9937 , the configuration error is also resolved. Thank you!

Reply:
Yes, it seem to be everywhere :) https://github.com/telefonicaid/iotagent-json/search?q=7896&unscoped_q=7896

NTC